languages/ruby.md

```diff
@@ -19,6 +19,21 @@ Alternatively, if your project uses Bundler, you can add the Solargraph gem to y
 gem 'solargraph', group: :development
 ```
 
+Solargraph have formatting and diagnostics disabled by default. We can tell Zed to enable them by adding the following to your `settings.json`:
+
+```json
+{
+   "lsp": {
+      "solargraph": {
+         "initialization_options": {
+            "diagnostics": true,
+            "formatting": true
+         }
+      }
+   }
+}
+```
+
```

Comment:
I thought about adding these under `Configuration` first, but I think most users expect this to be working out of the box, so I put it under `Setup` instead.

Reply:
Hey Kevin, I added you as a maintainer to this repo.  Are you able to merge this PR?

Reply:
I did. Thanks! 🙏